src/pages/bukime-budrus/patikima-informacija.js

```diff
@@ -29,6 +30,7 @@ const Page = ({ data }) => {
 
       {!!content && (
         <Constraint>
+          <Breadcrumb crumbs={[`Titulinis`, `Patikima informacija`]} />
```

Comment:
maybe it's better to move crumbs properties to separate const example `crumbsData` instead passing whole array